dotcom-rendering/src/components/BylineLink.tsx

```diff
@@ -140,13 +149,25 @@ export const BylineLink = ({ byline, tags, format }: Props) => {
 	return (
 		<>
 			{renderedTokens}
-			{renderingTarget === 'Apps' && soleContributor !== undefined ? (
-				<Island priority="critical">
-					<FollowWrapper
-						displayName={soleContributor.title}
-						id={soleContributor.id}
+			{renderingTarget === 'Apps' &&
+			!isHeadline &&
+			soleContributor !== undefined ? (
```

Comment:
We have a helper from `@guardian/libs`

```suggestion
			!isUndefined(soleContributor) ? (
```